sqlalchemy_dremio/base.py

```diff
@@ -0,0 +1,266 @@
+# sqlalchemy_dremio/base.py
+# Copyright (C) 2007-2012 the SQLAlchemy authors and contributors <see AUTHORS file>
+# Copyright (C) 2007 Paul Johnston, paj@pajhome.org.uk
+# Portions derived from jet2sql.py by Matt Keranen, mksql@yahoo.com
+#
+# This module is part of SQLAlchemy and is released under
+# the MIT License: http://www.opensource.org/licenses/mit-license.php
+
+"""
+Support for the Dremio database.
+
+
+"""
+from sqlalchemy import sql, schema, types, exc, pool
+from sqlalchemy.sql import compiler, expression
+from sqlalchemy.engine import default, base, reflection
+from sqlalchemy import processors
+
+class AcNumeric(types.Numeric):
```

Comment:
Probably can chuck all of these AcTypes and use the underlying sql types directly and map out the special cases using the Drill SA driver as a reference.
https://github.com/JohnOmernik/sqlalchemy-drill/blob/d213e26cd6b5ae07ad675e72d015ea4d7d03773b/sqlalchemy_drill/sadrill.py#L14